www/SimpleCameraPreview.js

```diff
@@ -2,6 +2,47 @@ var exec = require("cordova/exec");
 var PLUGIN_NAME = "SimpleCameraPreview";
 var SimpleCameraPreview = function () {};
 
+SimpleCameraPreview.videoInitialized = false;
+SimpleCameraPreview.videoCallback = null;
```

Comment:
```suggestion
```

I think `videoInitialized` is enough and storing `videoCallback` as global is not required